library/src/test/java/com/getstream/sdk/chat/Mother.kt

```diff
@@ -100,11 +100,14 @@ fun createMessage(
 		id: String = randomString(),
 		cid: String = randomString(),
 		text: String = randomString(),
-		createdAt: Date? = Date.from(Instant.now())
-): Message = Message(id, cid, text, createdAt = createdAt)
+		createdAt: Date? = Date.from(Instant.now()),
+		parentId: String? = null
+): Message = Message(id, cid, text, createdAt = createdAt, parentId = parentId)
 
 fun createMessageList(size: Int = 10) = (0..size).map { createMessage() }.toList()
 
+fun createThreadMessageList(size: Int = 10, parentMessageId: String = createMessage().id) = (0..size).map { createMessage() }.toList()
```

Comment:
`parentMessageId` is not used. Did you forget to use it?
On the other hand, because you are only to use the id of it, you don't need to create a complete message, you  can use `randomString()` and it will be faster while tests are run